moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -315,12 +324,14 @@ class Robot
 
   void updateState(const sensor_msgs::JointStateConstPtr& msg);
 
+  void updateCurrentState(const sensor_msgs::JointStateConstPtr& msg);
 
   // Robot internal objects
   std::map<std::string, TrajectoryInfo> traj_info_map_;
 
   // MoveIt objects
   mutable moveit::core::RobotStatePtr robot_state_;
```

Comment:
One says robot_state_ and other one is called current_robot_state_. Should we name them like online_robot_state_ and virtual_robot_state_?